src/sentry/notifications/notifications/digest.py

```diff
@@ -77,7 +81,9 @@ def get_subject(self, context: Mapping[str, Any] | None = None) -> str:
             # This shouldn't be possible but adding a message just in case.
             return "Digest Report"
 
-        return get_digest_subject(context["group"], context["counts"], context["start"])
+        # Use timezone from context if available (added by get_recipient_context)
+        timezone = context.get("timezone")
+        return get_digest_subject(context["group"], context["counts"], context["start"], timezone)
```

Comment:
Since `get_recipient_context` always returns a timezone (falls back to UTC) could we just do:

```suggestion
        return get_digest_subject(context["group"], context["counts"], context["start"], context["timezone")
```

or alternately if we're not sure just add the fallback

```suggestion
        return get_digest_subject(context["group"], context["counts"], context["start"], context.get("timezone", UTC))
```